portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,87 @@
+          French as a first language and I emigrated to the United States four years ago with the dream to become an engineer. 
+          I was completely overwhelmed at first because I spoke almost no English. It wasn't straightforward 
+          for me to understand and talk to people.
+          However, in the Summer of 2015, I started with English as a second language (ESL) at 
+          Waubonsee Community College.
+          I completed the program one year later, and I graduated with my Associate Degree in computer science in May 2019. Now I am a junior student studying computer science at 
+          the University of Illinois at Urbana Champaign. I ended up choosing computer science because of my passion for technology and the desire 
+          I have to solve complex problems. In 2015, I was nominated to emigrate to the United States and continue my education; my mother, however, was hesitant. At first, she did not want me to go. 
+          She thought it would be hard to succeed in another country where another language was spoken. I reminded her of all she and my father taught me, and I convinced her I could succeed. 
+          I ended up coming to the United States where I continue to work hard to pursue my dream of becoming a software engineer.</p>
+           <h2 style="font-size:300%;"> Unique Moment</h2>
+          <p style="font-size:200%;"> Before transferring to the University of Illinois at Urbana - Champaign,
+              I was a student at Waubonsee Community College, where I graduated with An Associate Degree in Engineering science. This picture of me was the day of my graduation. 
+              This moment is unique to me because three years from that day, I would not imagine getting to that point, looking at my English language.</p>
+          <img src="images/image3.jpeg"/>
+          <h2 style="font-size:300%;"> Unique Leadership Experience</h2>
+          <p style="font-size:200%;"> While at Waubonsee Community College, I had the opportunity to be part of the Student Government, 
+              where I met great people and serve as a Senator for three semesters.</p>
+          <img class="resize" src="images/image5.jpg"/>
+           <h2 style="font-size:300%;"> What I do during my free time!</h2>
+          <p style="font-size:200%;"> During my free time, I enjoy watching soccer and also giving back to my Community. I spent the past two spring breaks volunteering with non-profit organizations. 
+              In 2018, I spent my spring break with 13 other students in Louisville, Kentucky, volunteering with Habitat for Humanity in Richmond, Kentucky, building safe, affordable housing, and learning about substandard living conditions. 
+              In 2019, we did the same thing, but in Houston, Texas, with All Hands and Hearts in Houston, Texas, helping with Hurricane Harvey recovery. </p>
+          <h2 style="font-size:200%;"> 2018 and 2019 Pictures!</h2>
```

Comment:
One small thing is you have a lot of `style="font-size:200%"` and `style="font-size:300%"` CSS attributes here. It's not a big deal, but your code could be made more readable if you move these into your .css file. For example, since you want most of your paragraphs to have font-size 200%, you could add a section that is:
```
p {
  font-size: 200%;
}
```
And then keep the inline font-size styles for any that you want to override.

Reply:
Okay 👌